### PR TITLE
Auto compare feature results to master results

### PR DESCRIPTION
**Pull request by joseph-robertson**
## Pull Request Description

Adds:
* `workflow/tests/master` folder with latest:
  * `results.csv`
  * `results_ashrae_140.csv`
  * `results_hvac_sizing.csv` 
* `workflow/compare.rb`

The script exports csv files which diff current results against the latest results. The csv files are stored as `comparisons` artifacts.

Then, upon merging a PR into master, a github action is triggered that commits/replaces "base" (master) results with "feature" results.

## Checklist

Not all may apply:

- [ ] EPvalidator.xml has been updated
- [ ] Tests (and test files) have been updated
- [ ] Documentation has been updated
- [ ] Changelog has been updated
- [ ] `openstudio tasks.rb update_measures` has been run
- [ ] No unexpected regression test changes on CI

### CODE REVIEW

File: .github/workflows/config.yml [hunk 1]
```diff
@@ -16,8 +23,62 @@ jobs:
...
+            results
+          name: results
+
+      - name: Commit latest results
+        shell: bash        
+        run: |
+          git status
+          cp results/results.csv workflow/tests/master/results.csv
+          cp results/results_ashrae_140.csv workflow/tests/master/results_ashrae_140.csv
+          cp results/results_hvac_sizing.csv workflow/tests/master/results_hvac_sizing.csv
```
Comment (shorowit) [inline .github/workflows/config.yml:73]: Can you use a wildcard so that it operates on all results csv files?

File: workflow/compare.rb [hunk 2]
```diff
@@ -0,0 +1,86 @@
+require 'csv'
+
+folder = 'comparisons'
+files = ['results.csv', 'results_ashrae_140.csv', 'results_hvac_sizing.csv']
```
Comment (shorowit) [inline workflow/compare.rb:4]: Can you use a wildcard so that it operates on all results csv files?

File: workflow/compare.rb [hunk 3]
```diff
@@ -0,0 +1,86 @@
+require 'csv'
```
Comment (shorowit) [inline workflow/compare.rb:1]: Can you move this file to workflow/tests/compare.rb?

**Review by shorowit — COMMENTED** [review]
This is really great, just a few minor comments.

File: workflow/compare.rb (hunk 3 above)
Comment (joseph-robertson) [reply workflow/compare.rb:1]: I'll need to also remove this item from the Rakefile.

Comment (shorowit) [reply workflow/compare.rb]: Good point. I'm okay with that.

**Comment by shorowit** [comment]
Let's give it a shot!

**Pull request merged**